conda_smithy/configure_feedstock.py

```diff
@@ -384,7 +384,7 @@ def main(forge_file_directory):
     build_config.CONDA_PY = 10
 
     recipe_dir = 'recipe'
-    config = {'docker': {'image': 'condaforge/linux-anvil', 'command': 'bash'},
+    config = {'docker': {'image': 'condaforge/linux-anvil', 'command': 'bash -ex'},
```

Comment:
👍 

* `-e` - The well known abort on first error.
* `-x` - Expands and prints each command to `stdout`.

xref: http://tldp.org/LDP/abs/html/options.html#AEN19601